app/serializers/serialize_exercise_representations.rb

```diff
@@ -20,6 +20,8 @@ class SerializeRepresentation
     delegate :exercise, :track, to: :representation
 
     def call
+      user_ids = [representation.feedback_author_id, representation.feedback_editor_id].compact
+      user_handles = user_ids.present? ? User.where(id: user_ids).pluck(:id, :handle).to_h : {}
```

Comment:
This has the same issue. You're still doing a query for each representation. Any reason why you didn't use my suggestion?